core/server/master/src/test/java/alluxio/master/file/DefaultFileSystemMasterTest.java

```diff
@@ -1570,9 +1571,9 @@ private void startServices() throws Exception {
     mRegistry = new MasterRegistry();
     JournalFactory factory = new MutableJournal.Factory(new URI(mJournalFolder));
     mBlockMaster = new BlockMaster(mRegistry, factory);
-    mExecutorService =
-        Executors.newFixedThreadPool(2, ThreadFactoryUtils.build("FileSystemMasterTest-%d", true));
-    mFileSystemMaster = new FileSystemMaster(mRegistry, factory,
+    mExecutorService = Executors
+        .newFixedThreadPool(2, ThreadFactoryUtils.build("DefaultFileSystemMasterTest-%d", true));
+    mFileSystemMaster = new DefaultFileSystemMaster(mRegistry, factory,
```

Comment:
actually I have different opinion. The following test looks actually important to me.
```java
  @Test
  public void stop() throws Exception {
    mFileSystemMaster.stop();
    Assert.assertTrue(mExecutorService.isShutdown());
    Assert.assertTrue(mExecutorService.isTerminated());
  }
```
and this constructor is actually only serving this purpose for dependency injection.
```java
  public DefaultFileSystemMaster(MasterRegistry registry, JournalFactory journalFactory,
      ExecutorServiceFactory executorServiceFactory)
```
what do you think? I would think it worths leaving the code as is rather than using factory method